docs/more/dev-tools/tracing.rst

```diff
@@ -0,0 +1,62 @@
+########
+Tracing
+########
+
+Polar tracing shows logs of how a query is evaluated.
+
+********************
+Enabling Tracing
+********************
+
+Debug mode is enabled by setting an environment variable ``POLAR_LOG=1``
+
+It will print out ``[debug]`` messages during query evaluation that shows how the query is executed.
+Notable things include
```

Comment:
I'm not sure if this is the right way to get rst to show a literal colon
```suggestion
Notable things include:
```

Reply:
looks right rendered :+1: